website/docs/notes/concepts.md

```diff
@@ -119,4 +136,4 @@ The main aim of processors is to keep data processing pipelines as similar as po
 
 ## Sample List
 
-[SampleList](https://mmf.sh/api/lib/common/sample.html#mmf.common.sample.SampleList) has been inspired from BBoxList in maskrcnn-benchmark, but is more generic. All datasets integrated with MMF need to return a [Sample](https://mmf.sh/api/lib/common/sample.html#mmf.common.sample.Sample) which will be collated into `SampleList`. Now, `SampleList` comes with a lot of handy functions which enable easy batching and access of things. For e.g. `Sample` is a dict with some keys. In `SampleList`, values for these keys will be smartly clubbed based on whether it is a tensor or a list and assigned back to that dict. So, end user gets these keys clubbed nicely together and can use them in their model. Models integrated with Pythia receive a `SampleList` as an argument which again makes the trainer unopinionated about the models as well as the datasets. Learn more about `Sample` and `SampleList` in their [documentation](https://mmf.sh/api/lib/common/sample.html).
+[SampleList](https://mmf.sh/api/lib/common/sample.html#mmf.common.sample.SampleList) has been inspired from BBoxList in maskrcnn-benchmark, but is more generic. All datasets integrated with MMF need to return a [Sample](https://mmf.sh/api/lib/common/sample.html#mmf.common.sample.Sample) which will be collated into `SampleList`. Now, `SampleList` comes with a lot of handy functions which enable easy batching and access of things. For e.g. `Sample` is a dict with some keys. In `SampleList`, values for these keys will be smartly clubbed based on whether it is a tensor or a list and assigned back to that dict. So, end user gets these keys clubbed nicely together and can use them in their model. Models integrated with MMF receive a `SampleList` as an argument which again makes the trainer unopinionated about the models as well as the datasets. Learn more about `Sample` and `SampleList` in their [documentation](https://mmf.sh/api/lib/common/sample.html).
```

Comment:
Also, mention in tip that SampleList is basically a dict that works as collator.